### PR TITLE
Interface

### PR DESCRIPTION
**Pull request by ramico**
don't think we need to go fancy

### CODE REVIEW

**Comment by ramico** [comment]
interface

File: includes/class-wc-siftscience-options.php [hunk 1]
```diff
@@ -26,7 +26,8 @@ class WC_SiftScience_Options {
 		public static $guid = 'siftsci_guid';
 		public static $stats = 'siftsci_stats';
 		public static $stats_last_sent = 'siftsci_stats_last_sent';
-
+		//min order value input id Rjamleh
```
Comment (nabsul) [inline includes/class-wc-siftscience-options.php:29]: I think the variable name is clear, no need for the comment.

File: includes/class-wc-siftscience-options.php [hunk 2]
```diff
@@ -26,7 +26,8 @@ class WC_SiftScience_Options {
...
+		public static $min_order_value = 'min_order_value';
```
Comment (nabsul) [inline includes/class-wc-siftscience-options.php:30]: This value is used in the options database table. It should start with 'siftsci_' like all the others so it doesn't conflict with other plugins.

File: includes/class-wc-siftscience-admin.php [hunk 3]
```diff
@@ -207,7 +207,12 @@ private function get_settings() {
 					'User & Order Name Prefix',
 					'Prefix to give order and user names. '
 					. 'Useful when you have have multiple stores and one Sift account.' ),
+//the added field
+				$this->get_number_input( WC_SiftScience_Options::$min_order_value,
+					'Order Minimum value to Send',
+					 'Orders less than this value wont get sent unless the value is equal to 0', 0 ),
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:213]: It looks like there's an extra space messing with alignment.

File: includes/class-wc-siftscience-admin.php [hunk 4]
```diff
@@ -207,7 +207,12 @@ private function get_settings() {
 					'User & Order Name Prefix',
 					'Prefix to give order and user names. '
 					. 'Useful when you have have multiple stores and one Sift account.' ),
+//the added field
+				$this->get_number_input( WC_SiftScience_Options::$min_order_value,
+					'Order Minimum value to Send',
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:212]: Change to: Minimum order value for auto send

File: includes/class-wc-siftscience-admin.php [hunk 5]
```diff
@@ -207,7 +207,12 @@ private function get_settings() {
 					'User & Order Name Prefix',
 					'Prefix to give order and user names. '
 					. 'Useful when you have have multiple stores and one Sift account.' ),
+//the added field
+				$this->get_number_input( WC_SiftScience_Options::$min_order_value,
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:211]: Let's move this under the autosend checkbox

File: includes/class-wc-siftscience-admin.php [hunk 6]
```diff
@@ -207,7 +207,12 @@ private function get_settings() {
...
+					'Order Minimum value to Send',
+					 'Orders less than this value wont get sent unless the value is equal to 0', 0 ),
 
+//the checkbox code marker Rjamleh
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:215]: Remove marker comment

File: includes/class-wc-siftscience-admin.php (hunk 3 above)
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:213]: Change to:
Orders less than this value will not be automatically sent to sift. Set to zero to send all orders.

**Pull request merged**